codex/node.nim

```diff
@@ -559,8 +561,8 @@ proc onStore(
 
     return success()
 
-  without indexer =? manifest.protectedStrategy.init(
-    0, manifest.numSlotBlocks() - 1, manifest.numSlots).catch, err:
+  without indexer =? manifest.verifiableStrategy.init(
```

Comment:
This only needs to download the slot's blocks, not all the blocks.

Reply:
It's not downloading all the blocks, it's slicing the required blocks from the space of all blocks.